CSharp/BatchExplorer/Helpers/BasicDataProvider.cs

```diff
@@ -1,137 +1,151 @@
-﻿//Copyright (c) Microsoft Corporation
```

Comment:
I am not against reviewing and fixing both the CRLF issue _and_ the new feature that you added related to searching -- but it's also the most controversial change and we need to think about it more internally (which might take a few days), which is why it would help if that change (which happens to include the CRLF stuff) was in a separate pull request. That way we don't block your other good changes from going in while we discuss it.